test/passing/tests/exp_grouping.ml.ref

```diff
@@ -272,3 +272,16 @@ let _ =
     foo.fooooo <- Fooo.foo fooo foo.fooooo ;
     Fooo fooo
   end
+
+let _ =
+  match x with
+  | A -> begin
+    match B with A -> fooooooooooooo | B -> fooooooooooooo
+  end
```

Comment:
Would it be rendered as a single line if it fits ? I think we need this test case.

Reply:
It wasn't in my first commit, now it does and I added more tests to see what it looks like for combinations of `break-cases` and `exp-grouping`